Utils/Dataflow/095_datasetInfoAMI/amiDatasets.py

```diff
@@ -100,6 +102,14 @@ def init_ami_client(userkey='', usercert=''):
                          " create proxy.\n")
         raise DataflowException("Failed to initialise AMI client: certificate"
                                 " not provided or not found.")
+    try:
+        ami_client.execute('ListCommands')
+    except Exception as e:
+        sys.stderr.write("(ERROR) Failed to perform test command"
+                         " ListCommands. Are you sure you have"
+                         " a valid certificate?\nException: %s\n" % str(e))
+        raise DataflowException("Failed to perform test command"
+                                " ListCommands.")
```

Comment:
When the stage reaches this point, the log looks this way:
```
[dkb@aiatlas171 095_datasetInfoAMI]$ ./amiDatasets.py -m s
2020-05-21 14:25:14 (INFO) (ProcessorStage) Configuration parameters:
<...>
(ERROR) Failed to perform test command ListCommands. Are you sure you have a valid certificate?
Exception: [SSL] PEM lib (_ssl.c:2597)
Traceback (most recent call last):
  File "./amiDatasets.py", line 253, in <module>
    main(sys.argv[1:])
  File "./amiDatasets.py", line 58, in main
    stage.CONFIG['ami']['usercert'])
  File "./amiDatasets.py", line 111, in init_ami_client
    raise DataflowException("Failed to perform test command"
pyDKB.dataflow.exceptions.DataflowException: Failed to perform test command ListCommands.
```

It does not look very friendly (for me, at least):
1. `Exception: [SSL] ...`  is not prepended with any prefix, which makes it look like a part of Python-generated log (just like the traceback info following it), not a "handled" log message.
1. The `(ERROR)` log line, even with the `Exception: ...`, is confusing: *whose* command is this test one? what was tested with it? Traceback gives a clue -- the exception was thrown in `init_ami_client`, but it would be better if one could get the meaning of the error without diving into it.
  I mean -- even thought the message *does* give a hint that one might want to look at the certificate files, it does *not* explain what happened (and what to check about these files except for their existence).

Reply:
Fair points, added a prefix and a message about the test command being executed.